packages/vite-plugin-shopify/src/html.ts

```diff
@@ -145,6 +145,12 @@ export default function shopifyHTML (options: Required<Options>): Plugin {
 
 const viteTagDisclaimer = '{% comment %}\n  IMPORTANT: This snippet is automatically generated by vite-plugin-shopify.\n  Do not attempt to modify this file directly, as any changes will be overwritten by the next build.\n{% endcomment %}\n'
 
+// Disable Shopify theme-check linter at top of file
+const themeCheckDisable = '{% comment %}theme-check-disable{% endcomment %}\n'
```

Comment:
There are 2 offenses during dev; could we specify those? `AssetUrlFilters,RemoteAsset`

Reply:
updated with your feedback. thanks

Reply:
I would love your thoughts on not turning off these checks so we can catch the issue and then build production assets for deployment.